Makefile

```
@@ -248,14 +248,18 @@ test-nightly-server: build-nightly-server
 
 ### Check Rules: static checks, quality tools ###
 
-check: check-shell-scripts
+check: check-shell-scripts check-yaml
 .PHONY: check
-
 # rule requires shellcheck and find to run
 check-shell-scripts: $(filter $(ALT_BIN)%,$(SHELLCHECK))
 	$(SHELLCHECK) -P tests/ -eSC2181 -fgcc $$(find $(ROOT_DIR) -name "*.sh")
 .PHONY: check-shell-scripts
 
+
+check-yaml: $(filter $(ALT_BIN)%,$(YAMLLINT_CMD))
+	$(YAMLLINT_CMD) -c $(CURDIR)/.yamllint.yaml $(CURDIR) 
```

```suggestion
	$(YAMLLINT_CMD) -c $(CURDIR)/.yamllint.yaml $(CURDIR)
```

trailing whitespace to be removed.